CodeReview/src/commands/CFocus.cpp

```diff
@@ -61,41 +63,44 @@ bool CFocus::canInterpret(Visualization::Item*, Visualization::Item*,
 Interaction::CommandResult* CFocus::execute(Visualization::Item*, Visualization::Item*,
 				const QStringList&, const std::unique_ptr<Visualization::Cursor>&)
 {
+	focusStep({}, {}, {});
+	return new Interaction::CommandResult{};
+}
 
+bool CFocus::focusStep(Visualization::Item *, QKeySequence, Interaction::ActionRegistry::InputState)
+{
```

Comment:
Hmm, for that you should better keep using your current approach (and keep the include), but still use the target's scene for the line below.